### PR TITLE
update image path to quay

### PR DESCRIPTION
**Pull request by ashleyvjoy**
# Description
This PR addresses 2 changes:

- Update image path to quay.io
- Update module versions

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| https://github.com/dell/csm/issues/1435 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Build the image and deployed operator
- [x] Ran e2e tests successfully for Powerflex with replication, resiliency and verified images are pulled from quay

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ spec:
               name: RELATED_IMAGE_sdc
             - value: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
               name: RELATED_IMAGE_karavi-authorization-proxy
-            - value: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.12.0
+            - value: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.10.0
```
Comment (shefali-malhotra) [inline config/manager/manager.yaml:50]: any reason for downgrading the version?

File: tests/e2e/testfiles/minimal-testfiles/storage_csm_powerscale_replica.yaml [hunk 2]
```diff
@@ -28,4 +28,4 @@ spec:
           #  - name: "TARGET_CLUSTERS_IDS"
           #    value: "self"
         - name: dell-replication-controller-init
-          image: quay.io/dell/container-storage-modules/dell-replication-init:v1.0.0
+          image: dellemc/dell-replication-init:v1.0.0
```
Comment (shefali-malhotra) [inline tests/e2e/testfiles/minimal-testfiles/storage_csm_powerscale_replica.yaml:31]: Is this expected to point to docker.io

File: config/manager/manager.yaml (hunk 1 above)
Comment (ashleyvjoy) [reply config/manager/manager.yaml:50]: @shefali-malhotra The current available version is 1.9.0 and next release version should be 1.10.0
@satyakonduri

File: tests/e2e/testfiles/minimal-testfiles/storage_csm_powerscale_replica.yaml (hunk 2 above)
Comment (ashleyvjoy) [reply tests/e2e/testfiles/minimal-testfiles/storage_csm_powerscale_replica.yaml:31]: yes. dell-replication-init:v1.0.0 will not be released in quay. It exists in dockerhub

File: config/manager/manager.yaml (hunk 1 above)
Comment (satyakonduri) [reply config/manager/manager.yaml:50]: Yes, @ashleyvjoy, the upcoming version is 1.10.0.

**Review by francis-nijay — DISMISSED** [review]

**Review by meggm — APPROVED** [review]

**Review by chimanjain — APPROVED** [review]

**Pull request merged**